package.json

```diff
@@ -83,6 +80,8 @@
     "sqlite": "^5.1.1",
     "sqlite3": "^5.1.7",
     "ts-node": "10.9.2",
+    "ts-jest": "^29.2.5",
+    "tsx": "4.7.1",
```

Comment:
Why do we need tsx?

Reply:
It's used it `start` npm command

Reply:
I think the start command can be removed, it is not called anywhere.